tests/ja4-quic/test.yaml

```diff
@@ -0,0 +1,11 @@
+requires:
+  min-version: 7.0.0
+  files:
+    - rust/src/quic/ja4.rs
+
+checks:
+  - filter:
+      count: 11
+      match:
+        event_type: quic
+        quic.ja4: q13d0310h3_55b375c5d22e_cd85d2d88918
```

Comment:
How can we have `q` and `h` ? Is this not hexadecimal only ?

Reply:
Could you please add a README for these new tests ?

Reply:
> How can we have `q` and `h` ? Is this not hexadecimal only ?

No, it isn't. `q` means QUIC (in contrast to `t` which is TLS) and `h3` is part of the advertised ALPN data. See https://github.com/FoxIO-LLC/ja4/blob/main/technical_details/JA4.md for more details.

Reply:
> Could you please add a README for these new tests ?

Yeah, can do.